.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: ruby
+sudo: false
+cache:
+  bundler: true
+rvm:
+  - 2.5.0
+env:
+  global:
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+before_install:
+  - gem update --system
+  - gem install bundler
+before_script:
+  - bundle exec rake db:create db:setup
+  
```

Comment:
You need to start Redis here. Something like `services: redis-server`?

https://docs.travis-ci.com/user/database-setup/#Redis